ci/tox.ini

```diff
@@ -0,0 +1,46 @@
+[tox]
+skipsdist = True
+skip_missing_interpreters = True
+env_list = format, lint, k8s-ci
+
+[testenv]
+set_env =
+    PYTHONBREAKPOINT=pdb.set_trace
+    PY_COLORS=1
+passenv =
+    PYTHONPATH
+
+[testenv:format]
+description = Apply coding style standards to code
+deps = -r {toxinidir}/requirements-ci.txt
+commands =
+    licenseheaders -t {toxinidir}/.copyright.tmpl -cy -o 'Canonical, Ltd' -d {toxinidir}
+    isort {toxinidir} --profile=black
+    black {toxinidir} --verbose
```

Comment:
black and isort are python tools that are fine -- but they're slow.  We should consider using `ruff` where possible since it's written in rust, it's fast, and simpler to deal with..

but for now, this is fine